app/org/maproulette/controllers/api/TaskReviewController.scala

```diff
@@ -425,4 +428,77 @@ class TaskReviewController @Inject() (
     }
   }
 
+  /**
+    * Returns a list of challenges that have reviews/review requests.
+    *
+    * @param reviewTasksType  The type of reviews (1: To Be Reviewed,  2: User's reviewed Tasks, 3: All reviewed by users)
+    * @param tStatus The task statuses to include
+    * @param excludeOtherReviewers Whether tasks completed by other reviewers should be included
+    * @return JSON challenge list
+    */
+  def listChallenges(
```

Comment:
This function should go in a new TaskReviewController in the frameworks package so that we can start moving the rest to it eventually.